### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by danielochuba**
Build the Desktop version of the Portfolio, Using a responsive media Queries

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,15 +67,20 @@ <h5 class="supporting-text">I’am a Software Developer</h5>
 
                 <section>
                     <div class="works-section">
-                        <h1 class="title">My Recent Works</h1>
+                        <h1 class="title">My Recent Works </h1>
                         <hr class="rule">
+
```
Comment (RaoAkif) [inline index.html:72]: Your design is looking great, but the purpose of this project is to make a design as close to FIGMA design as possible.

| Your Design  | FIGMA Design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 2]
```diff
@@ -64,98 +92,122 @@ <h2 class="Title-post-1">Multi-Post Stories</h2>
...
+                                    
+                                    <div class="tags deskoff">
+                                        <div>
+                                            <ul class="tag-link ">
+                                                <li class="tag "><a href="https://www.html.org">html</a></li>
+                                                <li class="tag "><a href="https://getbootstrap.com/">bootstrap</a></li>
+                                                <li class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+                                            </ul>
+                                        </div>
+                                        <span class="see-project see-project-vis "><a href="">See project</a></span>
```
Comment (RaoAkif) [inline index.html:111]: It seems that this button has misaligned while adding some query. Try to make a design as close to FIGMA design as possible.

| Your Design  | FIGMA Design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 3]
```diff
@@ -64,98 +92,122 @@ <h2 class="Title-post-1">Multi-Post Stories</h2>
...
                 </section>
                 <section>
                     <div class="About">
-                        <h2 class="about-me">About me</h2>
-                        <span><p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></span>
-                        <span class="get-resume"><a href="">Get My Resume</a></span>
-                        <div class="image-2">
-
+                        
+                            <h2 class="about-me">About me</h2>
```
Comment (RaoAkif) [inline index.html:206]: It seems that this button has been misaligned while adding some query. I have tested your design between `760 - 1440` sizes, it's looking the same.

Try to make a design as close to FIGMA design as possible.

| Your Design  | FIGMA Design |
| ------------- | ------------- |
| image  | image  |

File: index.html [hunk 4]
```diff
@@ -64,98 +92,122 @@ <h2 class="Title-post-1">Multi-Post Stories</h2>
...
+                            <span><p class="description description-abt">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></span>
+                            <span class="get-resume"><a href="">Get My Resume</a></span>
+                            <div class="image-2">
+
```
Comment (RaoAkif) [inline index.html]: #### In the next section from the highlighted one:

The bullet points are different from the ones mentioned in the FIGMA design. Also look at the `Skill` word. It is pushed forward
Try to make a design as close to FIGMA design as possible.

| Your Design  | FIGMA Design |
| ------------- | ------------- |
| image | image  |

File: desktop.css [hunk 5]
```diff
@@ -0,0 +1,629 @@
...
+
+#more-text {
+  border: 0;
+  color: #000;
+  background: hsl(0, 11%, 95%);
+  font-style: normal;
+  font-family: 'Inter', Arial, Helvetica, sans-serif;
+  margin-left: -73%;
+}
+
```
Comment (RaoAkif) [inline desktop.css]: Try to make a design as close to FIGMA design as possible.

 Your Design:
 image
FIGMA Design:
image

**Review by RaoAkif — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️

Hi @danielochuba, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -44,15 +67,20 @@ <h5 class="supporting-text">I’am a Software Developer</h5>
 
                 <section>
                     <div class="works-section">
-                        <h1 class="title">My Recent Works</h1>
+                        <h1 class="title">My Recent Works </h1>
```
Comment (GedeonTS) [inline index.html:70]: - [OPTIONAL] Kindly make sure your work section is centered as per the Figma design:
- [OPTIONAL] Kindly make sure there is a hover effect to these cards as show in the design:

Screenshot from 2023-01-21 14-38-39

File: index.html [hunk 7]
```diff
@@ -203,7 +258,8 @@ <h2 class="description-2">I'm always interested in hearing about new projects,
                                 <input type="email" name="email-input" id="email-input" placeholder="    Email address" required>
                             </label><br>
                             <label for="more-text">
-                                <textarea name="" id="more-text" cols="30" rows="6" maxlength="500" placeholder="Enter text here" required></textarea>
+                                <textarea name="" id="more-text" cols="30" rows="6" maxlength="500" placeholder="Enter text here" class="mob" required></textarea>
+                                <textarea name="" id="more-text" cols="77" rows="6" maxlength="500" placeholder="Enter text here" class="desk" required></textarea>
                             </label>
```
Comment (GedeonTS) [inline index.html:263]: - [OPTIONAL] I think would be better to make sure the text area is aligned with the other inputs as in the Figma design

__After checking out the commit before the previous review I noticed that the errors were there before, that's why it's optional for now.__

- Your design:
Screenshot from 2023-01-21 15-31-30


- Figma:

Screenshot from 2023-01-21 14-40-27

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @danielochuba ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- Good use of Github flow :+1: 
## Optional suggestions :small_airplane: 
- [OPTIONAL]  Kindly make sure you have the future feature in your README as recommended by Microverse.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**